### PR TITLE
Create README

### PR DESCRIPTION
**Pull request by Shreyanand**
@suppathak @codificat Should we add more content or is this good for now? feel free to edit.

### CODE REVIEW

**Comment by codificat** [comment]
> @suppathak @codificat Should we add more content or is this good for now? 

the part about adapting LLMs needs polish. I would also add a basic README in the notebooks dir, if only to avoid the 404 that the current link would create.

> feel free to edit.

Happy to contribute to the above. Can you tick the box in the PR that allows modifications by maintainers?  I don't seem to be able to at the moment

File: README.md [hunk 1]
```diff
@@ -1,75 +1,31 @@
...
 
---------
+Traditional search engines rely on keywords to find relevant documents. However, this approach has limitations. It can be hard to know which keywords to use, and it may not capture the nuances of natural language. Large language models, on the other hand, can understand natural language queries and take into account the user's intent. This means that users can ask questions in a natural way, and the model will understand what they mean. For example, a user might ask "How many nodes can I have in my ROSA cluster?" and the model can provide the answer with links to relevant documentation.
 
-<p><small>Project based on the <a target="_blank" href="https://drivendata.github.io/cookiecutter-data-science/">cookiecutter data science project template</a>. #cookiecutterdatascience</small></p>
+In this README, we will explore how large language models can be used to improve product documentation search.
```
Comment (suppathak) [inline README.md:12]: Just a suggestion to add to the sentence above, 

"Our goal is to demonstrate the potential for leveraging foundation models to create an interactive, conversational interface that ROSA customers can use to access helpful tips and information, in addition to providing comprehensive technical documentation." 
Wdyt?

**Review by suppathak — COMMENTED** [review]
Thanks @Shreyanand . I just added a minor suggestion. lmkwyt?

**Comment by Shreyanand** [comment]
#21 is the editable version with these changes.

**Pull request closed without merging**